### PR TITLE
OPENEUROPA-0000: Documentation updates.

### PR DESCRIPTION
**Pull request by upchuk**

### CODE REVIEW

File: docs/architecture/third-party/drupal/content-architecture.md [hunk 1]
```diff
@@ -1 +1,14 @@
...
\ No newline at end of file
+# Content architecture
+
+The OpenEuropa project does not enforce a strict content architecture but provides corporate solutions used by the European Commission.
+
+## Content types
+
+The [OpenEuropa Content](https://github.com/openeuropa/oe_content) component provides a growing number of corporate content types intended to harmonize the content architecture across the sites of the European Commission.
+
+Moreover, it provides RDF-based support for the corporate vocabularies published by the European Publication Office (OP).
```
Comment (voidtek) [inline docs/architecture/third-party/drupal/content-architecture.md]: Can we add the link of the European Publication Office?
e.g. https://publications.europa.eu/en/home

File: docs/architecture/third-party/drupal/content-architecture.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+
+## Page building
+
+To support the building of pages with a variety of page-level elements, the [OpenEuropa Paragraphs](https://github.com/openeuropa/oe_paragraphs) component provides a number of Drupal paragraph types that can be used
+on any content type. Combined with the [OpenEuropa Theme](https://github.com/openeuropa/oe_theme), these are displayed using the [Europa Component Library (ECL)](https://ec.europa.eu/component-library) style guide.
```
Comment (voidtek) [inline docs/architecture/third-party/drupal/content-architecture.md]: Can we change the ecl link with https://github.com/ec-europa/europa-component-library?

File: docs/architecture/third-party/drupal/layout-architecture.md [hunk 3]
```diff
@@ -1 +1,6 @@
--- TO BE DEFINED --
\ No newline at end of file
+# Layout architecture
+
+The OpenEuropa project strictly follows the guidelines put forward by the [Europa Component Library (ECL)](https://ec.europa.eu/component-library).
```
Comment (voidtek) [inline docs/architecture/third-party/drupal/layout-architecture.md]: Can we change the ecl link with https://github.com/ec-europa/europa-component-library?

File: docs/development/ci-deployment-workflow.md [hunk 4]
```diff
@@ -1,7 +1,5 @@
-
 # CI Deployment Workflow
 
-OpenEuropa is using [Drone][1] for continuous integration and continuous delivery.
-For more information about Drone please check [Drone](docs/development/third-party/drone.md)]
+OpenEuropa is using [Drone](https://drone.io) for continuous integration and continuous delivery.
 
-[1]: https://drone.io
+You can see a default Drone pipeline configuration in the [Drupal module template repository](https://github.com/openeuropa/drupal-module-template/blob/master/.drone.yml).
```
Comment (voidtek) [inline docs/development/ci-deployment-workflow.md]: This paragraph is true for a drupal module, but for a site, it's different. Can we be more specific?
Can we add a paragraph concerning the server https://drone.fpfis.eu and the connexion with github?

File: docs/development/ci-deployment-workflow.md [hunk 5]
```diff
@@ -1,7 +1,5 @@
-
 # CI Deployment Workflow
```
Comment (voidtek) [inline docs/development/ci-deployment-workflow.md]: This file is not very clear. It's about the deployment of what?

File: docs/architecture/third-party/drupal/content-architecture.md (hunk 2 above)
Comment (upchuk) [reply docs/architecture/third-party/drupal/content-architecture.md]: Why?

File: docs/architecture/third-party/drupal/layout-architecture.md (hunk 3 above)
Comment (upchuk) [reply docs/architecture/third-party/drupal/layout-architecture.md]: Why?

File: docs/development/ci-deployment-workflow.md (hunk 4 above)
Comment (upchuk) [reply docs/development/ci-deployment-workflow.md]: I changed the link and added a small mention. We are talking about components here and not sites really.

File: docs/development/ci-deployment-workflow.md (hunk 5 above)
Comment (upchuk) [reply docs/development/ci-deployment-workflow.md]: Of the components

File: docs/architecture/third-party/drupal/content-architecture.md (hunk 2 above)
Comment (voidtek) [reply docs/architecture/third-party/drupal/content-architecture.md]: The link https://ec.europa.eu/component-library can change depending on the generator use.
But the Github is where the code and the documentation are based.

File: docs/architecture/third-party/drupal/layout-architecture.md (hunk 3 above)
Comment (voidtek) [reply docs/architecture/third-party/drupal/layout-architecture.md]: Response on the previous comment.

File: docs/development/ci-deployment-workflow.md (hunk 5 above)
Comment (voidtek) [reply docs/development/ci-deployment-workflow.md]: thanks

File: docs/development/ci-deployment-workflow.md (hunk 4 above)
Comment (voidtek) [reply docs/development/ci-deployment-workflow.md]: thanks.

File: docs/architecture/third-party/drupal/content-architecture.md (hunk 2 above)
Comment (ademarco) [reply docs/architecture/third-party/drupal/content-architecture.md]: Not really, that's actually the only unique URL among all of them as it's the project one.

**Pull request merged**